### PR TITLE
test(transport-test): implement `other call in progress` check for reaceive/send/call endpoints

### PR DESCRIPTION
**Pull request by mroz22**
concurrent receive/send/call old  vs new bridge behaves differently

a couple of not strictly related commits (f139b86, 1441e59, 2e920b5, 444f939),  cherry-picked into #13907. low hanging-ness guaranteed.

cc @szymonlesisz

### CODE REVIEW

File: packages/transport-test/e2e/bridge/bridge.test.ts [hunk 1]
```diff
@@ -226,4 +226,26 @@ describe('bridge', () => {
...
+            success: false,
+            error: 'session not found',
+            message: undefined,
+        });
+
+        expect([results[1], results[2]]).toIncludeAllPartialMembers([
+            { success: true, payload: { type: 'Features', message: expect.any(Object) } },
+            { success: false, error: 'other call in progress' },
+        ]);
+    });
```
Comment (mroz22) [inline packages/transport-test/e2e/bridge/bridge.test.ts]: + test enumerate and call

File: packages/transport/src/api/abstract.ts [hunk 2]
```diff
@@ -137,10 +146,51 @@ export abstract class AbstractApi extends TypedEmitter<{
     }
 
     protected unknownError<E extends AnyError>(err: Error, expectedErrors: E[]) {
+        console.log('unknownError', err);
```
Comment (szymonlesisz) [inline packages/transport/src/api/abstract.ts]: log

File: packages/transport-bridge/src/core.ts [hunk 3]
```diff
@@ -190,25 +192,27 @@ export const createCore = (apiArg: 'usb' | 'udp' | AbstractApi, logger?: Log) =>
         const { path } = sessionsResult.payload;
         logger?.debug(`core: call: retrieved path ${path} for session ${session}`);
 
-        const openResult = await api.openDevice(path, false, signal);
+        return api.runInIsolation({ read: true, write: true }, async () => {
```
Comment (szymonlesisz) [inline packages/transport-bridge/src/core.ts:191]: catch error

Comment (mroz22) [reply packages/transport-bridge/src/core.ts:191]: added

File: packages/transport/src/api/abstract.ts (hunk 2 above)
Comment (mroz22) [reply packages/transport/src/api/abstract.ts]: removed

**Review by szymonlesisz — APPROVED** [review]

**Comment by mroz22** [comment]
@trezor/qa - please pay attention to webusb and node-bridge on various platforms. Especially Windows where this was not tested very much.

@matejkriz  - just in case there are any troubles with the device connection in react native.

**Pull request merged**